vscode-client/snippets/snippets.json

```diff
@@ -0,0 +1,228 @@
+{
+    "shebang": {
+        "description": "[sh]hebang",
+        "prefix": "shebang",
+        "body": [
+            "#!/usr/bin/env ${1|bash,sh|}"
+        ]
+    },
+    "if": {
+        "description": "[if]",
+        "prefix": "if",
+        "body": [
+            "if ${1:command}; then",
+            "\t$0",
+            "fi"
+        ]
+    },
+    "if-else": {
+        "description": "[if] [e]lse",
+        "prefix": "if-else",
+        "body": [
+            "if ${1:command}; then",
+            "\t${2:echo}",
+            "else",
+            "\t$0",
+            "fi"
+        ]
+    },
+    "while": {
+        "description": "[wh]ile",
+        "prefix": "while",
+        "body": [
+            "while ${1:command}; do",
+            "\t$0",
+            "done"
+        ]
+    },
+    "until": {
+        "description": "[un]til",
+        "prefix": "until",
+        "body": [
+            "until ${1:command}; do",
+            "\t$0",
+            "done"
+        ]
+    },
+    "for": {
+        "description": "[fo]r",
+        "prefix": "for",
+        "body": [
+            "for ${1:variable} in ${2:list}; do",
+            "\t$0",
+            "done"
+        ]
+    },
+    "function": {
+        "description": "[fu]nction",
```

Comment:
Late comment: Does the `[fu]` naming here in the description make sense? Wasn't that to make the short prefixes easier to understand?

Reply:
And as the names are now rather descriptive, should be also remove the brackets from the ones where the prefix matches the name (e.g `echo` instead of `[echo]`)?

Reply:
It's my mistake, I'll gonna to correct mnemonics soon. :)

Reply:
PR has been created.